### PR TITLE
feat: 🇵🇰 pakistan feed

### PR DESCRIPTION
**Pull request by melMass**

### CODE REVIEW

File: src/pages/search/index.js [hunk 1]
```diff
@@ -316,16 +316,17 @@ export class Search extends Component {
     flag: false,
     lastId: undefined,
     tags: [
-      { id: 11, value: '🇺🇦 ukraine', label: 'ukraine' },
-      { id: 12, value: '🏳️‍🌈 tezospride', label: 'tezos pride' },
+      { id: 10, value: 'pakistan', label: '🇵🇰 pakistan' },
```
Comment (LeonNicholls) [inline src/pages/search/index.js]: These are used as aria-label values for accessibility. The emoticons don't work with the text-to-speech systems, so should be avoided. Just keep textual info for the labels.

Comment (melMass) [reply src/pages/search/index.js]: We could either use the value or another key, but the label/value was used the other way around. This also avoids emoticons check in ifs

Comment (melMass) [reply src/pages/search/index.js]: @NoRulesJustFeels fixed in the last commit

**Review by xat — APPROVED** [review]
don't have the time to properly review. but merch said, that he manually tested it. so merging it now.

**Pull request merged**